Languages/English/Keyed/AllKeyed_Help.xml

```diff
@@ -46,24 +46,24 @@ Section 1.1 - Physical properties
   <PRF_Shearer_HelpText>Automatically shears animals when they wander near the building. Store food as bait for the animals that they'll be drawn in to be sheared.</PRF_Shearer_HelpText>
   <PRF_Milker_HelpText>Automatically milks animals when they wander near the building. Store food as bait for the animals so that they'll be drawn in to be milked.</PRF_Milker_HelpText>
   <PRF_GenericAnimalHarvester_HelpText>Automatically harvests animals when they wander near the building. Store food as bait for the animals so that they'll be drawn in to be harvested.</PRF_GenericAnimalHarvester_HelpText>
-  <PRF_Slaughterhouse_HelpText>Automatically slaughters animals if they meet a specified criteria such as age, gender, or total quantity of animals. Also check out the Auto Meat Grinder and Animal stations for a truly automated farming experience.</PRF_Slaughterhouse_HelpText>
+  <PRF_Slaughterhouse_HelpText>Automatically slaughters animals if they meet specified criteria such as age, gender, or total quantity of animals. Also, check out the Auto Meat Grinder and Animal stations for a truly automated farming experience.</PRF_Slaughterhouse_HelpText>

```

Comment:
that confuses me. why put a comma after just one word?

Reply:
Seems this is a comma after the introductory phrase "Also". This is probably like comma after "Yes" or "No".

Reply:
seems odd but correct.